### PR TITLE
Bluetooth: Services: Refactor bluetooth services Kconfig

### PR DESCRIPTION
**Pull request by joerchan**
Refactor bluetooth services define. This avoids having to update a long
list of OR statements in CMakeLists.txt everytime a new service is
added. Follow pattern established by bluetooth mesh, this would allow us
to have a kconfig option for common services code as well, similar to
model_util.c

Signed-off-by: Joakim Andersson <joakim.andersson@nordicsemi.no>

### CODE REVIEW

**Comment by joerchan** [comment]
Context:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/77/files#r218098508

File: subsys/bluetooth/services/Kconfig [hunk 1]
```diff
@@ -4,7 +4,11 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-comment "BLE GATT Services"
+config BT_NORDIC_SERVICES
```
Comment (SebastianBoe) [inline subsys/bluetooth/services/Kconfig]: BT_NRF_SERVICES ?

File: subsys/bluetooth/services/Kconfig [hunk 2]
```diff
@@ -4,7 +4,11 @@
...
+	bool "nRF BT GATT Services support"
```
[the remark this answers is not in the document]
Comment (joerchan) [reply subsys/bluetooth/services/Kconfig]: Will do the same for MESH.

File: subsys/bluetooth/services/Kconfig (hunk 1 above)
Comment (joerchan) [reply subsys/bluetooth/services/Kconfig]: Yes, better, will update for MESH as well.

**Pull request merged**